Correct Molecule linting issues and remove flake8 linter

```shell
> molecule lint
INFO     default scenario test matrix: dependency, lint
INFO     Running default > dependency
Starting galaxy role install process
- downloading role 'mysql', owned by geerlingguy
- downloading role from https://github.com/geerlingguy/ansible-role-mysql/archive/3.3.0.tar.gz
- geerlingguy.mysql (3.3.0) was installed successfully
INFO     Dependency completed successfully.
WARNING  Skipping, missing the requirements file.
INFO     Running default > lint
COMMAND: set -e
yamllint .
ansible-lint
flake8

./molecule/default/converge.yml
  7:5       error    wrong indentation: expected 6 but found 4  (indentation)
```

--- a/molecule/default/converge.yml
+++ b/molecule/default/converge.yml
@@ -4,11 +4,11 @@
 
   vars:
     mysql_databases:
-    - name: "{{ prestashop_db_name }}"
-      collation: utf8mb4_general_ci
-      encoding: utf8mb4
-      replicate: 1
-      state: present
+      - name: "{{ prestashop_db_name }}"
+        collation: utf8mb4_general_ci
+        encoding: utf8mb4
+        replicate: 1
+        state: present
 
     mysql_users:
       - name: "{{ prestashop_db_user }}"

--- a/molecule/default/molecule.yml
+++ b/molecule/default/molecule.yml
@@ -4,10 +4,8 @@ dependency:
 driver:
   name: docker
 lint: |
-  set -e
   yamllint .
-  ansible-lint
-  flake8
+  ansible-lint .
 platforms:
   - name: instance
     image: "geerlingguy/docker-${MOLECULE_DISTRIBUTION:-ubuntu2004}-ansible:latest"

--- a/tasks/main.yml
+++ b/tasks/main.yml
@@ -2,7 +2,7 @@
 - name: check prestashop installation status
   stat:
     path: "{{ prestashop_root_path }}/{{ prestashop_check_file_name }}"
-  register: prestashop_is_installed
+  register: prestashop_status
 
 - name: create prestashop root directory
   file:

